src/main/groovy/com/instabot/data/model/user/User.groovy

```diff
@@ -168,6 +168,8 @@ class User {
         this.userStatus = userStatus
     }
 
+    // TODO NOOO! Never put such methods in Entities. Only getters and setters
+    // You can change it with a nice toString if you want.
```

Comment:
You mean?
 public String toString() {
        XmlMapper xmlMapper = new XmlMapper();
        String xml = xmlMapper.writeValueAsString(this)
    }